### PR TITLE
Review denis

### PR DESCRIPTION
**Pull request by popaula937**
- Small bug fixes

### CODE REVIEW

File: tvb_epilepsy/io/h5_writer.py [hunk 1]
```diff
@@ -399,6 +399,31 @@ def write_generic(self, object, folder, path):
...
+        :param object:
+        :param path:H5 path to be written
+        """
+        h5_model = convert_to_h5_model(object)
+
+        h5_model.add_or_update_metadata_attribute(self.H5_TYPE_ATTRIBUTE, "HypothesisModel")
+        h5_model.add_or_update_metadata_attribute(self.H5_SUBTYPE_ATTRIBUTE, object.__class__.__name__)
+
+        h5_model.write_to_h5(folder, path)
+
```
Comment (liadomide) [inline tvb_epilepsy/io/h5_writer.py]: These will go away, in the next merge (from branch review), right?

File: tvb_epilepsy/io/h5_reader.py [hunk 2]
```diff
@@ -384,5 +387,22 @@ def read_simulation_settings(self, path):
...
+        elif isequal_string(statistical_model_type, EpileptorStatisticalModels.ODESTATISTICAL_MODEL.value):
+            return EpileptorStatisticalModelBuilders.ODESTATISTICAL_MODEL_BUILDER.value.\
+                        initialize_from_statistical_model_dict(statistical_model).generate_model()
+        elif isequal_string(statistical_model_type, EpileptorStatisticalModels.STATISTICAL_MODEL.value):
+            return EpileptorStatisticalModelBuilders.STATISTICAL_MODEL_BUILDER.value.\
+                        initialize_from_statistical_model_dict(statistical_model).generate_model()
+        else:
+            raise_value_error(statistical_model_type + " is none of the available statistical model types!:\n" +
+                              str([_.value for _ in EpileptorStatisticalModelBuilders]))
+
```
Comment (liadomide) [inline tvb_epilepsy/io/h5_reader.py]: These will go away, in the next merge (from branch review), right?

File: tvb_epilepsy/service/model_inversion/statistical_models_builders.py [hunk 3]
```diff
@@ -28,43 +31,68 @@ class StatisticalModelBuilderBase(object):
...
+        statistical_model = self._set_attributes_from_dict(statistical_model)
+        model_config_dict = statistical_model.get("model_config", None)
+        if model_config_dict is None:
+            warning("Model configuration not found in statistical model read from file!" +
+                    "\nLeaving as it is: " + str(self.model_config))
+        if model_config_dict is not None:
+            setattr(self, "model_config",
+                    ModelConfigurationBuilder().build_model_from_model_config_dict(model_config_dict))
+        return self
+
```
Comment (liadomide) [inline tvb_epilepsy/service/model_inversion/statistical_models_builders.py]: These will go away, in the next merge (from branch review), right?

File: tvb_epilepsy/service/model_inversion/statistical_models_builders.py [hunk 4]
```diff
@@ -381,11 +409,17 @@ def generate_model(self):
...
                          str(time.time() - tic) + ' sec for model generation')
-        return model
\ No newline at end of file
+        return model
+
+
+class EpileptorStatisticalModelBuilders(Enum):
+    STATISTICAL_MODEL_BUILDER = StatisticalModelBuilder()
+    ODESTATISTICAL_MODEL_BUILDER = ODEStatisticalModelBuilder()
+    SDESTATISTICAL_MODEL_BUILDER = SDEStatisticalModelBuilder()
```
Comment (liadomide) [inline tvb_epilepsy/service/model_inversion/statistical_models_builders.py]: These will go away, in the next merge (from branch review), right?

Comment (popaula937) [reply tvb_epilepsy/service/model_inversion/statistical_models_builders.py]: Yes, to all of the above.

**Pull request merged**